department-of-veterans-affairs/abd-vro
# Github Actions: Send a "Failure" message to Slack on any SecRel failure

## What was the problem?
Failed SecRel scans were only being notified to one dedicated email address.

Associated tickets or Slack threads:
- #927 

## How does this fix it?[^1]
A failed SecRel scan will send a [slack message](https://dsva.slack.com/archives/C04CA47HV96/p1675298668567209) to the #benefits-vro-devops channel

## How to test this PR
- Introduce a known vulnerability into code, push it and mirror it to abd-vro-internal
- A message is sent to #benefits-vro-devops, detailing the branch, originator of the change and a link to the failed job.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
